app/models/manager_refresh/inventory/core.rb

```diff
@@ -0,0 +1,41 @@
+module ManagerRefresh::Inventory::Core
```

Comment:
hm, also this might go under automation and some under core? Since there will be shared ones and manager specific?